__tests__/ReactApp.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import { render, screen, cleanup } from '@testing-library/react';
+import App from '../client/src/components/App.jsx';
+
+
+beforeEach(() => {
+  render(<App />);
+});
+
+afterEach(() => {
+  cleanup();
+});
+
+test('Top level App component renders', () => {
+  const mainAppTestId = screen.getByTestId('app-main');
```

Comment:
The jest documentation recommends using testId infrequently, or as a last resort.  Seems like the most straight forward route to me, so I approve, just wanted to make sure you knew what the documentation recommended incase you haven't seen that.

Reply:
noted.